style.css

```diff
@@ -195,14 +194,13 @@ aside {
 .articleBoxLarge p {
     display: inline-block;
     float: left;
-    border: 1px solid green;
     height: 90%;
     width: 60%;
     margin: 10px;
     text-align: left;
-    background-color: RGBA(202,223,255,0.5);
+    background-color: RGBA(202,223,255,0.7);
```

Comment:
this might be silly but im used to seing it as `rgba(202, 223, 255, 0.7)` . So capitalization and spacing...